test/unit/components/VBtn/VBtn.spec.js

```diff
@@ -145,4 +145,35 @@ test('VBtn.js', ({ mount, compileToFunctions }) => {
 
     expect(wrapper.html()).toMatchSnapshot()
   })
+
+  it('should have v-btn--depressed class when using depressed prop', () => {
+    const wrapper = mount(VBtn, {
+      propsData: {
+        depressed: true
+      }
+    })
+
+    expect(wrapper.html()).toMatchSnapshot()
+  })
+
+  it('should have v-btn--flat class when using flat and depressed props', () => {
+    const wrapper = mount(VBtn, {
+      propsData: {
+        depressed: true,
+        flat: true
+      }
+    })
+
+    expect(wrapper.html()).toMatchSnapshot()
+  })
+
+  it('should have v-btn--outline and v-btn--depressed classes when using outline prop', () => {
+    const wrapper = mount(VBtn, {
+      propsData: {
+        outline: true
+      }
+    })
+
+    expect(wrapper.html()).toMatchSnapshot()
+  })
```

Comment:
These tests are useless IMHO, the path that adds classes is extremely basic and doesn't really need an entire snapshot for each and every combination of props.

Reply:
They are to increase % branch tests by testing all conditionals for `v-btn--depressed` class https://github.com/vuetifyjs/vuetify/blob/dev/src/components/VBtn/VBtn.js#L75
```javascript
'v-btn--depressed': (this.depressed && !this.flat) || this.outline
```
Please feel free to remove which tests you think is useless.

Reply:
You can just test if the class exists, no need to create full snapshot